packages/p2p/src/stores/sendbird-store.ts

```diff
@@ -466,6 +466,7 @@ export default class SendbirdStore extends BaseStore {
                 const msg_idx = this.chat_messages.findIndex(msg => msg.id === msg_identifier);
                 if (channel_message.isUserMessage()) {
                     this.replaceChannelMessage(msg_idx, 1, convertFromChannelMessage(channel_message));
+                    this.messages_ref?.current?.scrollTo(0, this.messages_ref.current.scrollHeight);
```

Comment:
everytime user sends a message, scroll to the bottom of the chat here instead of using useEffect to listen to chat_messages being updated